gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     istio: ingressgateway
     release: {{ .Release.Name }}
 spec:
+  strategy:
```

Comment:
Is this something we want in production ? 

Can you use a kustomize template ? That seems to be the consensus for changing k8s-specific things.

Reply:
We patch it anyway, so we don't need it. We will remove it.

But why would you this *NOT* in production? Otherwise, you would have a downtime during update, right?

Reply:
We should absolutely do rolling update - but not with hardcoded parameters you have.

That's why I suggested kustomize template or values.yaml

Reply:
> We should absolutely do rolling update - but not with hardcoded parameters you have.
> 
> That's why I suggested kustomize template or values.yaml

I agree that this is something you ultimately want to have configurable. But a hardcoded rolling update would still be better than the default, right?